environment_python_3_9.yml

```diff
@@ -42,7 +42,7 @@ dependencies:
   - psycopg2=2.8.6
   - pytest=6.2.4
   - pytest-cov=3.0.0
-  - python=3.7.10
+  - python=3.8.5
```

Comment:
This should be 3.9.11?

Reply:
Looks like I missed this, I'll get on it and merge.